### ISSUE TITLE
JS

### ISSUE THREAD

**Issue by EvgeniyChuprov** [fatfutor/range-slider#3]
Переписать js на классы ES6.
Код должен соответствовать стандартам fsd и стайлгайду AirBnB, в том числе и по оформлению. Никаких var, ++ и так далее.
Поставить ESlint и настроить его на правила AirBnB.
+ Почему код слайдера не полностью на ts?

**Comment by fatfutor** [comment]
на js только тесты, где-то читал, что это разрешено

**Comment by EvgeniyChuprov** [comment]
![Снимок экрана от 2020-02-03 13-22-38](https://user-images.githubusercontent.com/36810364/73645394-48da8b00-4688-11ea-917a-e7fe1afa2514.png)
Нет, я про это. Я так понял это js скомпиленный из ts?

**Comment by fatfutor** [comment]
> ![Снимок экрана от 2020-02-03 13-22-38](https://user-images.githubusercontent.com/36810364/73645394-48da8b00-4688-11ea-917a-e7fe1afa2514.png)
> Нет, я про это. Я так понял это js скомпиленный из ts?

да, в гитигнор не поставил сразу, поэтому компилируемые файлы ушли на гитхаб.

**Comment by EvgeniyChuprov** [comment]
Конструкторы надо минимизировать.
![Снимок экрана от 2020-02-11 13-24-40](https://user-images.githubusercontent.com/36810364/74228653-f0cc0600-4cd1-11ea-8e53-a72e71703426.png)

По стандартам airbnb, для приведения к числу нужно использовать Number:
![Снимок экрана от 2020-02-11 13-27-36](https://user-images.githubusercontent.com/36810364/74228831-4bfdf880-4cd2-11ea-9f40-2ac677d9933a.png)

Большое количество параметров нужно передавать объектом, а после деструктуризировать.
![Снимок экрана от 2020-02-11 13-29-05](https://user-images.githubusercontent.com/36810364/74229043-a5febe00-4cd2-11ea-8369-78d7dd3c701e.png)

**Comment by fatfutor** [comment]
и для каждого параметра писать интерфейс?

**Comment by EvgeniyChuprov** [comment]
Написать интерфейс для аргумента который содержит все параметры.

**Comment by fatfutor** [comment]
> Написать интерфейс для аргумента который содержит все параметры.

вроде так и сделал

**Comment by EvgeniyChuprov** [comment]
Проглядел исправление)
